OlinkAnalyze/R/Olink_plate_randomizer.R

```diff
@@ -151,20 +151,42 @@ generatePlateHolder <- function(n.plates,n.spots,n.samples, PlateSize){
 
 #' Randomly assign samples to plates
 #'
-#' Samples can be randomly assigned to plates using base::sample with an option to keep Subjects on the same plate.  Olink Data Science no longer recommends forced balanced randomization considering other clinical variables. 
+#' Generates a scheme for how to plate samples with an option to keep subjects on the same plate.
+#'
+#' Variables of interest should if possible be randomized across plates to avoid confounding with potential plate effects. In the case multiple samples per subject (e.g. in longitudinal studies), Olink recommends keeping each subject on the same plate. This can be achieved using the SubjectColumn argument.
+#'
+#' The generated plating scheme can be visualized using the function \code{\link{olink_displayPlateLayout}}.
+#' How well the relevant variables are randomized can be visualized using the function \code{\link{olink_displayPlateDistributions}}.
 #' @param Manifest tibble/data frame in long format containing all sample ID's. Sample ID column must be named SampleID.
 #' @param PlateSize Integer. Either 96 or 48. 96 is default.
 #' @param SubjectColumn (Optional) Column name of the subject ID column. Cannot contain missings. If provided, subjects are kept on the same plate.
 #' @param iterations Number of iterations for fitting subjects on the same plate.
-#' @param available.spots Integer. Number of wells available on each plate. Maximum 40 for T48 and 88 for T96. Can also take a vector equal to the number of plates to be used indicating the number of wells available on each plate.
+#' @param available.spots Integer. Number of wells available on each plate. Maximum 40 for T48 and 88 for T96. Takes a vector equal to the number of plates to be used indicating the number of wells available on each plate.
```

Comment:
Change Integer to Numeric? Not sure if the class changes with a vector or not.

Reply:
Either numeric or integer works here (does it ever make a difference in R? I don't think I ever distinguish between the two :) ). Since numeric is the most common type, I agree that we should probably write that instead

Reply:
Maybe it would also be good to add an example on how to use this argument?